charts/gatekeeper/templates/_helpers.tpl

```diff
@@ -47,8 +47,9 @@ Output post install webhook probe container entry
 - name: webhook-probe-post
   image: "{{ .Values.postInstall.probeWebhook.image.repository }}:{{ .Values.postInstall.probeWebhook.image.tag }}"
   imagePullPolicy: {{ .Values.postInstall.probeWebhook.image.pullPolicy }}
-  args:
+  command:
```

Comment:
`/manifest_staging` changes will get promoted to `/charts` in the next release, we can revert changes to `/charts`